### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mogarg**
To help us review your pull request, please consider providing an overview of the following:
* What is the type of the change (bug fix, feature, documentation and etc.) ?
* What are the current behavior and expected behavior, if this is a bugfix ?
* What are the steps required to reproduce the bug, if this is a bugfix ?
* What is the current behavior and new behavior, if this is a feature change or enhancement ?
* [Optional] Why is the new behavior better than the current behavior, if this is a feature change ?

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -2,33 +2,38 @@
...
 This is the comdb2 test suite.
 
+### Prerequisites
+
 Some of the tests require non standard tools to be installed, so here is a 
 list of packages needed to run all the tests:
 
 `apt-get install bash coreutils jq tcl-dev`
 
-Clustered tests also need ssh-client and ssh-server setup.
```
Comment (riverszhang89) [inline tests/README.md:10]: I don't think ssh-client and ssh-server here refer to specific software packages. I'd rephrase the sentence to "Cluster tests also require SSH."

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: Success &#10003;.<br>Smoke testing: Success &#10003;.<br>Cbuild submission: <a href='http://comdb2.s3-website-us-east-1.amazonaws.com/tests/94165aa8/robomark.txt'>Error &#x26A0; </a>.<br>Regression testing: <b>4/401 tests failed</b> &#x26A0;.</p><p>The first 10 failing tests are: <br>simple_timepart_reptimeout_generated<br>sc_timepart_logicalsc_generated<br>simple_timepart<br>datetime</p><p><a href='http://comdb2.s3-website-us-east-1.amazonaws.com/tests/94165aa8/detail.txt'>Detail</a> | <a href='http://comdb2.s3-website-us-east-1.amazonaws.com/tests/94165aa8/logs.tar.gz'>Log</a> | <a href='http://ec2-34-202-216-70.compute-1.amazonaws.com/restricted/index.php?id=2730'>Resubmit</a></p>
